examples/cms-wordpress/components/hero-post.js

```diff
@@ -20,7 +20,10 @@ export default function HeroPost({
         <div>
           <h3 className="mb-4 text-4xl lg:text-6xl leading-tight">
             <Link as={`/posts/${slug}`} href="/posts/[slug]">
-              <a className="hover:underline">{title}</a>
+              <a
+                className="hover:underline"
+                dangerouslySetInnerHTML={{ __html: title }}
+              />
```

Comment:
Why is the title added like this? can it contain html?

Reply:
Yes...titles in WordPress *can* contain HTML.

![screenshot](https://dl.dropbox.com/s/rfi3b5f5tp5z880/Screenshot%202020-05-28%2008.35.28.png?dl=0)

Reply:
😮